composer.json

```diff
@@ -103,6 +106,11 @@
         "sort-packages": true,
         "allow-plugins": false
     },
+    "extra": {
+        "google/apiclient-services": [
```

Comment:
do we really need that?

Reply:
can't we just specify that we only need `google/apiclient-services`? I'm not huge fan of automated scripts like this cleanup as they like to break and that might break entire repo

Reply:
yes it will work, i remove it